pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_HDAWG8.py

```diff
@@ -447,19 +481,35 @@ def upload_waveform_realtime(self, w0, w1, awg_nr: int, wf_nr: int =1):
         self._realtime_w0 = w0
         self._realtime_w1 = w1
 
+        print('wf_nr', wf_nr)
```

Comment:
This print statement should be removed. Otherwise we completely spam the terminal in daily use.